src/Storages/MergeTree/ReplicatedMergeTreeSink.cpp

```diff
@@ -161,7 +263,33 @@ size_t ReplicatedMergeTreeSink::checkQuorumPrecondition(const ZooKeeperWithFault
     return replicas_number;
 }
 
-void ReplicatedMergeTreeSink::consume(Chunk chunk)
+std::vector<String> getBlockIDVec(BlockWithPartition & block, String partition_id)
```

Comment:
Not sure, but name is unclear...
```suggestion
std::vector<String> getHashesForBlocks(BlockWithPartition & block, String partition_id)
```